frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -78,13 +83,43 @@ export const MilestonesEditor = () => {
     setRoadmapsVersionsLocal(roadmapsVersions);
   }, [roadmapsVersions]);
 
+  // Checks if tasks in the tasks list have been given ratings, returns only tasks with ratings from everyone involved
+  const checkRatings = useCallback(
+    (uncheckedTasks: Task[]) => {
+      const ratedTasks: Task[] = [];
+      const developers = allUsers?.filter((user) => user.type === 279715);
```

Comment:
```suggestion
      const developers = allUsers?.filter((user) => user.type === RoleType.Developer);
```

Löytyy `redux/user/types`